blocks/folder-blocks/infinite-canvas/Item.tsx

```diff
@@ -143,36 +137,12 @@ export const Item = ({
                 options={relevantBlockOptions || []}
               />
             </div>
-            {BlockComponent && block ? (
+            {BlockComponent && block && (
               <div className={tw(`w-full flex-1`)}>
                 <div className={tw(`scaled-down`)}>
-                  <BlockComponent
-                    {...(blockProps || {})}
-                    block={block}
-                    path={path} // soon to be deprecated, leaving for backwards compatibility
-                    context={{
-                      path,
-                    }}
-                    content={contents}
-                    onRequestGitHubData={async (...args) => {
-                      // catch API errors
-                      try {
-                        return await blockProps.onRequestGitHubData(...args);
-                      } catch (e) {
-                        console.error(e);
-                      }
-                    }}
-                  />
+                  <BlockComponent block={block} context={{ path }} />
                 </div>
               </div>
-            ) : (
```

Comment:
iirc this supports items that are just plain text (eg. not a Block) - is the plan to get rid of that?

Reply:
Plain text items are handled [earlier](https://github.com/githubnext/blocks-examples/pull/60/files#diff-80fba83fc6fa5341b63f9fb7cf2ab3780df77946fe272c54dd6869065a5d14d6L106-L119). This code shows the file contents as text before the block is loaded; I decided to drop it because loading is a lot faster now and it ends up being a FOUC (flash of unblockular content).

Reply:
😂 perfect!